src/phoenix/server/api/types/Span.py

```diff
@@ -306,39 +309,6 @@ def default(self, obj: Any) -> Any:
         return super().default(obj)
```

Comment:
We don't need to address this here, but we should probably have this default to a safe encoding.

Reply:
Good call.